app/build.gradle

```diff
@@ -137,6 +148,7 @@ dependencies {
     implementation "io.reactivex.rxjava2:rxandroid:$rxandroid_version"
     implementation "com.jakewharton.rxrelay2:rxrelay:$rxrelay_version"
     implementation "com.tomasznajda.ktx:rxjava2:$rxjavaktx_version"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
Do we need this?

Reply:
No, not really. No idea how it sneaked in ...